src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -8026,6 +8026,793 @@ .maxstack  4
+                    // (23,9): error CS8352: Cannot use variable 's' in this context because it may expose referenced variables outside of their declaration scope
+                    //         s.Deconstruct(out x2, out y2);
+                    Diagnostic(ErrorCode.ERR_EscapeVariable, "s").WithArguments("s").WithLocation(23, 9),
+                    // (23,9): error CS8350: This combination of arguments to 'Program.Deconstruct(ReadOnlySpan<int>, out int, out R)' is disallowed because it may expose variables referenced by parameter 's' outside of their declaration scope
+                    //         s.Deconstruct(out x2, out y2);
+                    Diagnostic(ErrorCode.ERR_CallArgMixing, "s.Deconstruct(out x2, out y2)").WithArguments("Program.Deconstruct(System.ReadOnlySpan<int>, out int, out R)", "s").WithLocation(23, 9));
+            }
+            else
+            {
+                comp.VerifyEmitDiagnostics(
+                    // (13,9): error CS8352: Cannot use variable '(x1, y1) = s' in this context because it may expose referenced variables outside of their declaration scope
+                    //         (x1, y1) = s; // 1
+                    Diagnostic(ErrorCode.ERR_EscapeVariable, "(x1, y1) = s").WithArguments("(x1, y1) = s").WithLocation(13, 9),
+                    // (13,20): error CS8350: This combination of arguments to 'Program.Deconstruct(ReadOnlySpan<int>, out int, out R)' is disallowed because it may expose variables referenced by parameter 's' outside of their declaration scope
+                    //         (x1, y1) = s; // 1
+                    Diagnostic(ErrorCode.ERR_CallArgMixing, "s").WithArguments("Program.Deconstruct(System.ReadOnlySpan<int>, out int, out R)", "s").WithLocation(13, 20),
+                    // (14,9): error CS8352: Cannot use variable 's' in this context because it may expose referenced variables outside of their declaration scope
+                    //         s.Deconstruct(out x1, out y1); // 2
+                    Diagnostic(ErrorCode.ERR_EscapeVariable, "s").WithArguments("s").WithLocation(14, 9),
+                    // (14,9): error CS8350: This combination of arguments to 'Program.Deconstruct(ReadOnlySpan<int>, out int, out R)' is disallowed because it may expose variables referenced by parameter 's' outside of their declaration scope
+                    //         s.Deconstruct(out x1, out y1); // 2
+                    Diagnostic(ErrorCode.ERR_CallArgMixing, "s.Deconstruct(out x1, out y1)").WithArguments("Program.Deconstruct(System.ReadOnlySpan<int>, out int, out R)", "s").WithLocation(14, 9));
+            }
+        }
+
+        [WorkItem(64448, "https://github.com/dotnet/roslyn/issues/64448")]
+        [Theory]
+        [InlineData(LanguageVersion.CSharp10)]
+        [InlineData(LanguageVersion.CSharp11)]
+        public void Deconstruct_05(LanguageVersion languageVersion)
```

Comment:
Same feedback here.

Reply:
Removed 5 (the equivalent Span<T> case was 6).